include/aws/iot/Mqtt5Client.h

```diff
@@ -173,6 +173,22 @@ namespace Aws
                 const Crt::Io::TlsContextPkcs11Options &pkcs11Options,
                 Crt::Allocator *allocator = Crt::ApiAllocator()) noexcept;
 
+            /**
+             * Sets the builder up for MTLS, using a PKCS#12 file for private key operations.
+             *
+             * @param hostName - AWS IoT endpoint to connect to
+             * @param pkcs12_file - The PKCS12 file to use
+             * @param pkcs12_password - The password for the PKCS12 file
+             * @param allocator - memory allocator to use
+             *
+             * @return Mqtt5ClientBuilder
+             */
+            static Mqtt5ClientBuilder *NewMqtt5ClientBuilderWithMtlsPkcs12(
+                const Crt::String hostName,
+                const char *pkcs12_file,
```

Comment:
Split file and password into a separate struct maybe?  Also, no raw c-strings

Reply:
I moved the struct to `MqttCommon.h` and used that in both MQTT311 and MQTT5.